### ISSUE TITLE
Typescript doc visible for aliased imports, but not for single ones.

### ISSUE THREAD

**Issue by waderyan** [microsoft/TypeScript#11099]
_From @Elarcis on September 21, 2016 7:35_
- VSCode Version: 1.5.2
- OS Version: Windows 7 Service Pack 1
- Typescript version: 2.0.2

Steps to Reproduce:
1.  have a `myClass.ts` file exporting a `MyClass` class. Write some `/** */` doc for that class.
2.  In another file, try the following two ways of importing identifiers:

``` Typescript
import { MyClass } from 'myModule';
MyClass // When hovering MyClass, the written doc is not visible.
```

``` Typescript
import * as myModule from 'myModule';
myModule.MyClass // When hovering MyClass, the written doc appears.
```

It seems logical that the doc should appear in both cases, is this due to Typescript 2?

_Copied from original issue: Microsoft/vscode#12360_

**Comment by waderyan** [comment]
_From @chrmarti on September 21, 2016 16:42_

Can reproduce with the given example and TypeScript 1.8 in todays insider build.

**Comment by waderyan** [comment]
Repro'd with latest TypeScript release bits - release-2.0.5 branch. 

Here is code to copy and paste. 

test.ts
```typescript
import { MyClass } from './myModule';
MyClass // When hovering MyClass, the written doc is not visible.

import * as myModule from './myModule';
myModule.MyClass // When hovering MyClass, the written doc appears.
```

myModule.ts
```typescript
/**
 * Some documentation
 */
export class MyClass {

}
```

sending to TypeScript language service repo.

**Comment by aluanhaddad** [comment]
This is currently a huge painpoint. Improving this would be a very big win.

**Comment by rozzzly** [comment]
I'd also like to point out that default imports don't get the (js)doc for that export either. 
```typescript
import SomeDefaultExport from './FooBar';
```
Hovering over `SomeDefaultExport` in both the import declaration as well as it's uses in code will just show "`import SomeDefaultExport`".

I'm not at my desktop at the moment, so I haven't tested this, but imagine the same problem would exist with imports like this:
```typescript
import { Foo as Bar } from './FooBar';
```
I'll update this post when I'm able to test that. 

My experience with this bug are in `typescript`, but I would imagine there would be the issue with `javascript` (that uses `es2015` imports). I'll test that out too.

**Comment by dasilvacontin** [comment]
I've lost quite some time thinking I was doing type exports / imports wrong due to not seeing the docs on hover. 😅

What's the status on this? Waiting for someone from the community to work on it?

**Comment by ambroselittle** [comment]
I'm the one who reported https://github.com/Microsoft/vscode/issues/28753. You can see the repro easily there. 

This _seems_ like something basic that should just work, especially consider that props work fine (and that seems like it'd be harder to support). :D But anyways, please fix/include the class/component-level docs inclusion soon!

**Comment by DanielRosenwasser** [comment]
Is this a duplicate of #5515?

**Comment by Andarist** [comment]
The reported code works OK today - the documentation is displayed. cc @jakebailey

**Comment by jakebailey** [comment]
Yeah, this was a dupe of #5515; fixed in #19489.
